R/tm_g_distribution.R

```diff
@@ -1160,6 +1170,48 @@ srv_distribution <- function(id,
       modal_title = "R Code for distribution",
       code_header = "Distribution"
     )
+
+    ### REPORTER
+    if (with_reporter) {
+      card_fun <- function(comment) {
+        card <- teal.reporter::TealReportCard$new()
+        card$set_name("Association plot")
+        card$append_text("Association plot", "header2")
+        card$append_text("Filter State", "header3")
+        card$append_fs(datasets$get_filter_state())
+        card$append_text("Main Element", "header3")
+        if (input$tabs == "Histogram") {
+          card$append_plot(dist_r(), dim = pws1$dim())
+        } else if (input$tabs == "QQplot") {
+          card$append_plot(qq_r(), dim = pws2$dim())
+        }
+        card$append_text("Statistics table", "header3")
+        card$append_table(teal.code::chunks_get_var("summary_table", chunks = common_code_chunks()))
```

Comment:
When I run exploratory in teal.gallery, I got `Warning in chunks$get(var = var) :
  "summary_table" does not exist in the chunks.`.

Reply:
`tryCatch` handles this now.